### PR TITLE
Limited FPS in F&R routing mode to reduce energy consumption

### PR DESCRIPTION
**Pull request by rokuz**

### CODE REVIEW

File: drape_frontend/frontend_renderer.cpp [hunk 1]
```diff
@@ -1011,6 +1011,14 @@ void FrontendRenderer::Routine::Do()
     frameTime = timer.ElapsedSeconds();
     timer.Reset();
 
+    // Limit fps in following mode.
+    double const kFrameTime = 1.0 / 30.0;
```
Comment (gardster) [inline drape_frontend/frontend_renderer.cpp]: constexpr

**Comment by bykoianko** [comment]
LGTM

**Comment by gardster** [comment]
LGTM после правки const

**Comment by rokuz** [comment]
@gardster PTAL

**Comment by gardster** [comment]
LGTM

**Pull request merged**